health-services/ExerciseSampleCompose/app/src/main/java/com/example/exercisesamplecompose/data/ExerciseClientManager.kt

```diff
@@ -66,6 +66,15 @@ class ExerciseClientManager @Inject constructor(
         }
     }
 
+
+    private var DISTANCE_THRESHOLD: Double = 0.0
```

Comment:
If these are private and can only be updated together, I'd make a data class called Thresholds with two vals: distance and duration, then change the signature of updateGoals to updateGoals(newThresholds: Thresholds)

Woul also make the private var thresholds lowercase as the uppercase implies constant?

Reply:
duration should probably be a Duration type. Prefer non primitive time values.

Reply:
Done!